### PR TITLE
[REF] marketing automation: move getting started

### PR DESCRIPTION
**Pull request by samueljlieber**
Task: [#3699212](https://www.odoo.com/web#id=3699212&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

This PR restructures the Marketing Automation directory to remove the `getting_started/*` sub-directory. This restructure is laying the foundation for future improvements to this app's documentation.

```
getting_started.rst --> marketing_automation.rst
getting_started/first_campaign.rst --> marketing_automation.rst
getting_started/* --> marketing_automation/*
getting_started/* --> marketing_automation/*
getting_started/* --> marketing_automation/*
```

### CODE REVIEW

**Comment by samueljlieber** [comment]
Updated :doc: path on `testing_running.rst` in https://github.com/odoo/documentation/commit/ff38935ac49d454d88e41e7e2b41c4221e2ebdd7

**Comment by samueljlieber** [comment]
> Just a quick thing— the toctree isn't rendering correctly on `marketing_automation.rst`. Can you take a look?

Hi @StraubCreative, thank you for the review! After some testing, this is how the Page TOC formats when there is a subdirectory below it, and not just individual files. So in this case the `marketing_automation/advanced/understanding_metrics` line is causing the TOC to format like this. I agree it looks kinda off! I am swapping to use a seealso since it is a little cleaner 👍
Screenshot 2024-02-07 at 9 29 52 AM

**Comment by StraubCreative** [comment]
> After some testing, this is how the Page TOC formats when there is a subdirectory below it

Gross 😅 We should change it + touch up the styles anyway, they could use some love.

File: content/applications/marketing/marketing_automation.rst [hunk 1]
```diff
@@ -1,17 +1,126 @@
...
+   - :doc:`marketing_automation/advanced/understanding_metrics`
 
 .. toctree::
    :titlesonly:
 
-   marketing_automation/getting_started
+   marketing_automation/target_audience
+   marketing_automation/workflow_activities
+   marketing_automation/testing_running
    marketing_automation/advanced
```
Comment (StraubCreative) [inline content/applications/marketing/marketing_automation.rst:126]: Do we think having a sub-directory called `advanced` is necessary?
Why not just have it at the same level as all the other docs as this also solves the toctree display issue?

What do you think?

Comment (samueljlieber) [reply content/applications/marketing/marketing_automation.rst:126]: I agree, my plans for improving this documentation do not invlove an `/advanced` directory, so I am happy to remove this as well and move `understanding_metrics.rst` to the same level as the others 👍

**Comment by samueljlieber** [comment]
In https://github.com/odoo/documentation/commit/22fb8d17a18b2b6c0a7e56edffe4ba2bfbfe09be I removed the `/advanced` directory and moved `understanding_metrics.rst` to the same level as the other docs.

**Review by StraubCreative — APPROVED** [review]
Seems good, thanks Sam 👍 

@robodoo r+

**Comment by StraubCreative** [comment]
@robodoo retry

**Comment by samueljlieber** [comment]
@robodoo retry

**Pull request closed without merging**